plugins/varnish/varnish5_

```diff
@@ -346,22 +348,22 @@ my %ASPECTS = (
 	},
 	'transfer_rates' => {
 		'title' => 'Transfer rates',
-		'order' => 's_resp_bodybytes s_resp_hdrbytes',
+		'order' => 's_resp_hdrbytes s_resp_bodybytes',
 		'args' => '-l 0',
 		'vlabel' => 'bit/s',
 		'values' => {
 			's_resp_hdrbytes' => {
 				'type' => 'DERIVE',
 				'label' => 'Header traffic',
-				'draw' => 'STACK',
+				'draw' => 'AREA',
```

Comment:
Hi,

I had made a mail explaining a bit more what I did here and I had sent it to the last person what had maintained varnish5_ @kjetilho 

On request_rates the change of color is required, the changes are far deeper than color changes.
The graph is the oposite of what it was so it would be posible to visualize passed and missed requests.  Instead of having them stacked on top of hits, here I stack hits (big figure) on top of passed and missed (small numbers)

transfer_rates, in the same logic, lets have small numbers to be the first ones stacked, big ones on top.

having small figures stacked on top of big ones (like cache hits) makes them impossible to visualise.  Hit rate can go from 1 to 200 in the course of a day, if one stacks on top something that varies between 0.5 and 3, the frange it creates on top of huge mountains isn't readable, and in practice it's the passed and miss we want to monitor, not the hits (hits aren't painful to backend and the hit-rate graph gives us what we need to know on hits, over time one wants to keep an eye on the evolution over time of 'pass' and 'miss')

Perhaps you want to have a look at the result
low trafic
https://munin.adm.ofil.fr/localdomain/localhost.localdomain/varnish5_request_rate.html
https://munin.adm.article3.fr/localdomain/localhost.localdomain/varnish5_request_rate.html


more serious trafic
https://munin.adm.welovetennis.fr/localdomain/localhost.localdomain/varnish5_request_rate.html

Reply:
> Is this change intentional? Or did you mean AREASTACK?

not sure I'm getting your question.
the plugin used AREA to define first graph and STACK for those that pile on top.
Di I miss something ?

Reply:
hi,
Looks like I had miss-understood `sess_conn` and working at fixing this.

I'll need a bit more test time and will come back with a new pull request.

Reply:
> the plugin used AREA to define first graph and STACK for those that pile on top.

Maybe it helps to simplify the code: you can also use `AREASTACK` for *all* fields. The old distinction between the first and the following fields is not necessary anymore.

> I'll need a bit more test time and will come back with a new pull request.

Feel free to recycle this PR by force-pushing into it. Or just close this one and open a new one.
Thanks!